src/liboslexec/llvm_util.cpp

```diff
@@ -1902,10 +1896,6 @@ void
 LLVM_Util::setup_new_optimization_passes(int optlevel, bool target_host)
 {
 #ifdef OSL_LLVM_NEW_PASS_MANAGER
-#    if OSL_LLVM_VERSION <= 110
-#        error "New pass manager not supported in LLVM 11 and earlier"
-#    endif
```

Comment:
This can't be removed yet, since it's <= 11 and not < 11.

Reply:
I'll revert that, but curiously, the "oldest everything" CI test passed using LLVM 11. That's because this doesn't technically matter -- we only turn on the new pass manager when LLVM version >= 16, and this check merely ensures that if it's on, we're using at least 11. So in a sense, it's redundant (other than to ensure that we don't change the logic about when we use the new pass manager to a version too early).

Reply:
Good catch, by the way, thanks for looking carefully at this.

Reply:
Pushed an update